ghc-lib-gen/src/GhclibgenOpts.hs

```diff
@@ -76,7 +76,8 @@ ghcFlavorOpt = option readFlavor
 
 readFlavor :: ReadM GhcFlavor
 readFlavor = eitherReader $ \case
+    "ghc-8.10.1" -> Right Ghc8101
     "ghc-8.8.1" -> Right Ghc881
     "da-ghc-8.8.1" -> Right DaGhc881
     "ghc-master" -> Right GhcMaster
-    flavor -> Left $ "Failed to parse ghc flavor " <> show flavor <> " expected ghc-8.8.1 or da-ghc-8.8.1"
+    flavor -> Left $ "Failed to parse ghc flavor " <> show flavor <> " expected ghc-8.8.1, da-ghc-8.8.1 or ghc-8.10.1"
```

Comment:
Why is `ghc-master` not listed as an expected input?

Reply:
Good point. Just an oversight - I'll fix it. (actually, even then it's an approximation. You can pass a SHA in that position too)